flixel/addons/editors/tiled/TiledTileSet.hx

```diff
@@ -59,61 +61,139 @@ class TiledTileSet
 		// internal
 		else 
 		{
-			var node:Fast = source.node.image;
-			imageSource = node.att.source;
-			
-			var imgWidth = Std.parseInt(node.att.width);
-			var imgHeight = Std.parseInt(node.att.height);
-			
-			name = source.att.name;
-			
-			if (source.has.tilewidth) 
-			{
-				tileWidth = Std.parseInt(source.att.tilewidth);
-			}
-			if (source.has.tileheight) 
-			{
-				tileHeight = Std.parseInt(source.att.tileheight);
-			}
-			if (source.has.spacing) 
-			{
-				spacing = Std.parseInt(source.att.spacing);
-			}
-			if (source.has.margin) 
-			{
-				margin = Std.parseInt(source.att.margin);
-			}
-			
-			// read properties
-			properties = new TiledPropertySet();
-			for (prop in source.nodes.properties) {
-				properties.extend(prop);
-			}
-			
-			// read tiles properties
-			tileProps = new Array<TiledPropertySet>();
-			
-			for (node in source.nodes.tile)
+			if (!source.hasNode.image)
 			{
-				if (!node.has.id)
+				//tileset with several images
+				var node:Fast = source.node.tile;
+				imageSource = "";
+				
+				name = source.att.name;
+				
+				if (source.has.tilewidth) 
```

Comment:
Again, there seems to be a fair amount of redundancy here (this section is quite similar to the one for "normal tileset").